qemu/tests/nested_kvm_unit_test.py

```diff
@@ -0,0 +1,39 @@
+import logging
+import os
+
+from virttest import error_context
+from avocado.utils import process
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Run nested relstead tests in kvm-unit-test test suite
+
+    1) Clone kvm-unit-tests test suite from repo
+    2) Compile test suite
+    3) Run nested related test cases on host
+    """
+
+    kvm_unit_test_dir = "/tmp/kvm_unit_test_dir"
+    mkdir_cmd = "mkdir -p %s" % kvm_unit_test_dir
+    rmdir_cmd = "rm -rf %s" % kvm_unit_test_dir
+
+    if os.path.exists(kvm_unit_test_dir):
+        process.system(rmdir_cmd)
+    process.system(mkdir_cmd)
+
+    clone_cmd = params["clone_cmd"] % kvm_unit_test_dir
+    process.system(clone_cmd, shell=True)
+    compile_cmd = params["compile_cmd"] % kvm_unit_test_dir
+    process.system(compile_cmd, shell=True)
```

Comment:
Is it works without `shell=True`? If yes, better to remove it.

Reply:
Removed for line 27 and keep for line 29.